maas-region/src/charm.py

```diff
@@ -264,8 +274,25 @@ def _update_ha_proxy(self) -> None:
                             [],
                         )
                     ],
-                }
+                },
             ]
+            if self.config["tls_mode"] == "termination":
+                data.append(
+                    {
+                        "service_name": "agent_service",
+                        "service_host": "0.0.0.0",
+                        "service_port": MAAS_PROXY_PORT,
+                        "servers": [
+                            (
+                                f"{app_name}-{self.unit.name.replace('/', '-')}",
+                                self.bind_address,
+                                MAAS_HTTP_PORT,
+                                [],
+                            )
+                        ],
+                    }
+                )
+            # TODO: Implement passthrough configuration
```

Comment:
raise an exception if this mode can be selected

Reply:
opting with removing "passthrough" as a valid option